src/wallet/wallet.h

```diff
@@ -1105,6 +1106,18 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
 
     /** Implement lookup of key origin information through wallet key metadata. */
     bool GetKeyOrigin(const CKeyID& keyid, KeyOriginInfo& info) const override;
+
+    std::map<std::string, CNamePendingData> namePendingMap;
+
+    bool PendingNameFirstUpdateExists(const std::string &name);
+    bool WritePendingNameFirstUpdate(
+            const std::string &name,
+            const valtype &tx,
+            const uint256 &triggerTxid,
+            const valtype &triggerName,
+            const int32_t &triggerDepth);
+    bool ErasePendingNameFirstUpdate(const std::string &name);
+    bool GetPendingNameFirstUpdate(const std::string &name, CNamePendingData *data=nullptr);
```

Comment:
This function is never used for now (probably it will be used later, though).  I suggest to remove it now and add it only in a follow-up PR when you actually use it.

Reply:
While these don't appear to be used in jeremy's branch, they are part of my branch and are used over at `managenames0.14`. These will certainly be needed once the pending functionality is complete.